code-challenges.js

```diff
@@ -34,3 +43,42 @@ var hand1 = [5, 5, 5, 3, 3]
 // Expected output: true
 var hand2 = [5, 5, 3, 3, 4]
 // Expected output: false
+
+ 
+ const fullHouse = (array) => {
+     if (threeOfAKind === true && pair === true)
+        return false;
+        // if threeOfAKind === true && pair === true
+}
+  const pair = (array) => {
+        for (let i = 0; i < array.length; i++) {
+            let count = 1;
+            for (let j = i + 1; j < array.length; j++) {
+                if (array[i] == array[j])
+                    count++;
+            }
+            if (count == 2)
+                return true;
+        }
+        return false;
+    }
+ 
+ 
+ const threeOfAKind = (array) => {
+        for (let i = 0; i <= 3; i++) {
+            let count = 1;
+            for (let j = i + 1; j < array.length; j++) {
+                if (array[i] == array[j])
+                    count++;
+            }
+            if (count == 3)
+                return true;
+        }
+        return false;
+    }
+
+console.log(fullHouse(hand1))
+console.log(fullHouse(hand2))
+
+// Console log is returning undefined for this one :(. Had a real hard time with this one. First thought to use array.from(new Set.Array) to find duplicates in the array, but couldn't figure out how to check for three of a kind and pairs. 
+// The logic above was from google and I don't quite understand it. Could we go over this problem in class? 
```

Comment:
this was a difficult problem so its understandable! i know that so far, joe and danny have some pretty interesting solutions to this. maybe reach out and have them share ^.^